src/Stripe.net/Infrastructure/Urls.cs

```diff
@@ -36,6 +36,8 @@ internal static class Urls
 
         public static string Subscriptions => BaseUrl + "/subscriptions";
 
+        public static string SubscriptionItems => BaseUrl + "/subscription_items";
+
```

Comment:
I am working towards eliminating this file. The urls are going directly in the service - see the StripeSubscriptionItemService